react/features/base/lib-jitsi-meet/functions.native.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+
+import { loadScript } from '../util';
+import logger from './logger';
+
+export * from './functions.any';
+
+const { JavaScriptSandbox } = NativeModules;
+
+/**
+ * Loads config.js from a specific remote server.
+ *
+ * @param {string} url - The URL to load.
+ * @returns {Promise<Object>}
+ */
+export async function loadConfig(url: string): Promise<Object> {
+    try {
+        const configTxt = await loadScript(url, 2.5 * 1000 /* Timeout in ms */, true /* skipeval */);
+        const configJson = await JavaScriptSandbox.evaluate(`${configTxt}\nJSON.stringify(config);`);
```

Comment:
@saghul sorry for stupid question this must have been raised many times, but why do we need to evaluate config.js? Why is it not enough to just parse it?

Reply:
Not a stupid question! What we get after doing the fetch is a text file. We need a JS object. At the time of this writing, there is other executable code in config.js which helps genrate the `config` variable: https://meet.jit.si/saghulHq/config.js (loot at the top).

The only way to get the `config` object is to execute the JS file and get its result.